src/Kunstmaan/GeneratorBundle/Generator/PagePartGenerator.php

```diff
@@ -144,18 +149,40 @@ private function generateResourceTemplate()
     private function generateSectionConfig()
     {
         if (count($this->sections) > 0) {
-            $dir = $this->bundle->getPath() . '/Resources/config/pageparts/';
+            if ($this->isSymfony4()) {
+                $dir = $this->bundle->getPath().'/../config/kunstmaancms/pageparts/';
```

Comment:
```suggestion
                $dir = $this->container->getParameter('kernel.project_dir') . '/config/kunstmaancms/pageparts/';
```

To keep the code inline with the check in [`KunstmaanGenerateCommand`](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7b48b9c1d0bcecca7970011e8ef46d818d3a9ebb/src/Kunstmaan/GeneratorBundle/Command/KunstmaanGenerateCommand.php#L351)